content/docs/recipes/category-tables.md

```diff
@@ -78,7 +78,7 @@ As with the [External Foreign Keys](/recipes/external-foreign-keys/) recipe, ref
 
 In a Category Table Resource, the field referenced by the `valueField` property `MUST` validated with `"required": true` and `"unique": true` field constraints. Similarly, when `labelField` is specified, the field it references `MUST` be of type `string` and be validated with the `"unique": true` field constraint.
```

Comment:
Are we sure we want to require that `labelField` be unique, i.e., that the mapping between `valueField` and `labelField` be one-to-one? I can see a few use cases for permitting a many-to-one mapping between `valueField` and `labelField`.